app/components/Input/index.tsx

```diff
@@ -8,10 +10,17 @@ type InputProps = {
   onChange?: (e: any) => void
   onKeyDown?: (e: any) => void
   onBlur?: (e: any) => void
+  multiline?: boolean
 }
-const Input = ({className, ...props}: InputProps) => {
+
+const Input = ({className, multiline, ...props}: InputProps) => {
   const classes = ['input', className].filter((i) => i).join(' ')
 
+  if (multiline) {
+    // Use createElement to pass direct element prop to AutoHeight
+    return createElement(AutoHeight, {element: 'textarea', className: classes, ...props})
```

Comment:
whoa, what's this? 👀  I will look into this ~tomorrow, but this code looks.. unexpected

Reply:
fixed in https://github.com/StampyAI/stampy-ui/pull/918/commits/ac614c0395494620e2e74dd08c3fbfb0ff8cf5ef

![Screenshot 2025-04-10 at 9 46 10](https://github.com/user-attachments/assets/b557c79e-561e-42e7-81c1-7904e28d276e)